showcase/src/Navigators/Stories.elm

```diff
@@ -0,0 +1,88 @@
+module Navigators.Stories exposing (stories, update)
+
+import UI.NavigationContainer as Nav
+import Element
+import Msg
+import PluginOptions exposing (defaultWithoutMenu)
+import Return as R exposing (Return)
+import UI.Checkbox as Checkbox
+import UI.Internal.Basics exposing (ifThenElse)
+import UI.Text as Text
+import UIExplorer exposing (storiesOf)
+import Utils exposing (goToDocsCallToAction, iconsSvgSprite, prettifyElmCode, storyWithModel)
+
+import Navigators.Model as Stories
+import Navigators.Msg as Stories
+
+update : Navigators.Msg -> Navigators.Model -> Return Navigators.Msg Navigators.Model
+update msg model =
+    case msg of
+        Navigators.ForNav newValue ->
+
+            let
+                ( newState, subCmd ) =
+                    Nav.stateUpdate subMsg model.navState
+            in
+            ( { model | state = newState }
+            , Cmd.map Navigators.ForNav subCmd )
+
+
+stories renderConfig =
+    storiesOf
+        "Checkboxes"
+        [ navigatorDemo renderConfig ]
+
+
+boxDemo renderConfig =
+    let
+
+        container  =
+            { title = "Vrau" 
+            , content = Nav.contentSingle Element.none
+            , dialog = Nothing
+            , hasMenu = False
+            }
+        view { navigatorsStories } =
+            Nav.navigator ForNav
+                navState
+                (toNavContainer appConfig context >> Nav.containerMap ForPage)
+                --|> Nav.withMenuPages (menuPageList appConfig.locale page)
```

Comment:
maybe remove this comment

Reply:
Oh boy, I totally forgot that I haven't finished that page :o hahaha, that's my bad, this is a test I was doing.